openshift/templates/dev/dev-secret.json

```diff
@@ -0,0 +1,20 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {},
+    "items": [
+        {
+            "kind": "Secret",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "tfrs-postgresql",
+                "creationTimestamp": null
+            },
+            "data": {
```

Comment:
Can you connect with the other devops people if this encryption is good enough for storing in the repo?